README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
```

Comment:
Include information about what 'task dev' does, and what the user should expect after running it, to provide context for the process.